commands/utils/validate.js

```diff
@@ -365,6 +365,27 @@ module.exports = validate_config = function (lt_config, validation_configs) {
       });
     }
 
+    //validate if dedicated proxy field contains expected value
+    if ("dedicated_proxy" in lt_config["run_settings"]) {
+      if (![true, false].includes(lt_config["run_settings"]["dedicated_proxy"])) {
+        reject("Error!! boolean value is expected in dedicated_proxy key");
+      }
+    }
+
+    //validate if dedicatedproxy and tunnel are not passed simultaniously
+    if (lt_config["run_settings"]["dedicated_proxy"]==true && lt_config["run_settings"]["geo_location"] !=""){
```

Comment:
Are we not also checking types here in comparison? Can we do `===` in the changes instead of `==`?
And use `!==` instead if `!=`.

Reply:
No it is not required

Reply:
@japneetlambdatest , why arent we checking type also? This might lead to any unexpected errors in js.

Reply:
Types are already checked in code above that is why it is not required here